tests/release/README.md

```diff
@@ -0,0 +1,46 @@
+# Release e2e tests
+
+Contains E2E tests related to [Release Service](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/release).
+
+This document will detail the e2e tests included for the release suite.
+
+## e2e tests
+### 1. Good Path (release.go)
```

Comment:
maybe we say this is the manual good path and 2 is the automated good path? Since we seem to manually create a release CR in this one